src/main/scala/au/com/onegeek/sbtdotenv/SbtDotenv.scala

```diff
@@ -80,7 +80,8 @@ object SbtDotenv extends AutoPlugin {
     state.log.debug(s"looking for .env file: ${baseDirectory}/${fileName}")
     val dotEnvFile: File = new File(s"${baseDirectory}/${fileName}")
     parseFile(dotEnvFile).map { environment =>
-      state.log.info(s".env detected (fileName=${fileName}). About to configure JVM System Environment with new map: $environment")
+      state.log.info(s".env detected (fileName=${fileName}). About to configure JVM System Environment with new map")
+      state.log.debug(s"New map: $environment")
```

Comment:
maybe add a comment here why this is at `debug`-level, otherwise we might unconsciously change it back.

Reply:
I've added a comment stating that, feel free to consider if it's appropriate

Reply:
I've added a comment stating that, feel free to consider if it's appropriate